Changes

```diff
@@ -377,47 +377,47 @@ OCaml 4.07
   (David Allsopp and Damien Doligez)
 
 - GPR#1526: install the debug and instrumented runtimes
-  (lib{caml,asm}run{d,i}.a)
+  (lib{caml,asm}run{d,i}.a).
   (Gabriel Scherer, reminded by Julia Lawall)
 
-- GPR#1563: simplify implementation of LSRINT and ASRINT
+- GPR#1563: simplify implementation of LSRINT and ASRINT.
   (Max Mouratov, review by Frédéric Bour)
 
 - GPR#1644: remove caml_alloc_float_array from the bytecode primitives list
-  (it's a native code primitive)
+  (it's a native code primitive).
   (David Allsopp)
 
-- GPR#1701: fix missing root bug in GPR#1476
+- GPR#1701: fix missing root bug in GPR#1476.
   (Mark Shinwell)
 
-- GPR#1752: do not alias function arguments to sigprocmask (Anil Madhavapeddy)
+- GPR#1752: do not alias function arguments to sigprocmask. (Anil Madhavapeddy)
 
 - GPR#1753: avoid potential off-by-one overflow in debugger socket path
-  length (Anil Madhavapeddy)
+  length. (Anil Madhavapeddy)
 
 ### Tools:
 
 - MPR#7643, GPR#1377: ocamldep, fix an exponential blowup in presence of nested
-  structures and signatures, e.g. "include struct … include(struct … end) … end"
+  structures and signatures, e.g."include struct … include(struct … end) … end"
```

Comment:
I think this is better with the space.

Reply:
Indeed - it’s also consistent with other instances